lib/test.js

```diff
@@ -88,6 +89,20 @@ function Test (options) {
   Readable.apply(this, options)
 }
 
+Test.prototype.beforeEach = function (fn) {
+  this.on('beforechild', fn)
```

Comment:
I think that having `mocha`ish methods that don't behave identically to their namesakes introduces a bit of confusion. I'm a big fan of gradually splitting this functionality into its own module that has feature parity with mocha.